lambda/templates/ecr_scan_notifications_lambda.json.tpl

```diff
@@ -0,0 +1,24 @@
+{
+  "Version": "2012-10-17",
+  "Statement": [
+    {
+      "Effect": "Allow",
+      "Action": [
+        "logs:CreateLogStream",
+        "logs:PutLogEvents"
+      ],
+      "Resource": [
+        "arn:aws:logs:eu-west-2:${account_id}:log-group:/aws/lambda/tdr-ecr-scan-notifications-mgmt",
```

Comment:
Should the environment "mgmt" be hardcoded?

Reply:
It can be because it's only in one environment but it's bad practice. I've passed the environment in.